src/async_fuse/memfs/s3_node.rs

```diff
@@ -531,6 +538,13 @@ impl<S: S3BackEnd + Sync + Send + 'static> Node for S3Node<S> {
 
     /// Decrease node open count
     fn dec_open_count(&self) -> i64 {
+        if self.open_count.load(Ordering::Acquire) == 0 {
+            warn!(
+                "dec_open_count() found open_count is 0, ino={}, name={}",
+                self.get_ino(),
+                self.get_name()
+            );
+        }
         debug_assert!(self.open_count.load(Ordering::Acquire) > 0);
```

Comment:
There is a gap in the atomic operation between `warn` and `debug_assert`, is it OK?

Reply:
It's OK. Lately ,the s3node is not thread safe since each s3node is owned by one operation.